src/Faithlife.Build/DotNetClassicTool.cs

```diff
@@ -60,7 +60,13 @@ public sealed class DotNetClassicTool
 
 		var packagesPath = Environment.GetEnvironmentVariable("NUGET_PACKAGES") ??
 			Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".nuget", "packages");
-		var packagePath = Path.Combine(packagesPath, packageName.ToLowerInvariant(), packageVersion);
+
+		// Find a matching package. If using a specific version, there will be exactly one match.
+		// If using floating versions, theyere may be multiple matches, so take the latest semantic version.
+		var packagePath = Directory.GetDirectories(Path.Combine(packagesPath, packageName.ToLowerInvariant()), packageVersion)
+			.OrderByDescending(fullPath => Version.Parse(Path.GetFileName(fullPath)))
```

Comment:
This will crash if the pattern matches any prerelease packages, e.g. `1.2.3-beta`. [Use NuGet.Versioning?](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#where-nugetversion-diverges-from-semantic-versioning)

Reply:
Weird.  I can get it to throw a `FormatException` in LinqPad with this, but when I tested with `StyleCop.Analyzers` `1.2.0-beta.435` seemed to Parse through just fine.  Maybe I was testing the exact version case? I really didn't want to bring in another package but that seems like the best way

Reply:
Fixed in https://github.com/Faithlife/FaithlifeBuild/pull/54/commits/6c268c88f63c04ad3ef84bd00033a0b9593455e8